### PR TITLE
New Commit

### PR DESCRIPTION
**Pull request by nicks9188**

### CODE REVIEW

File: jhgkh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Author: nicks9188 <nicks9188@gmail.com>
+Date:   Wed Feb 20 19:58:45 2019 +0530
+
+    corrected address
+
+[33mcommit ba4b42eaf817c7d8db8c07b5bdae9d631fbdf5d6[m
+Author: nicks9188 <nicks9188@gmail.com>
+Date:   Wed Feb 20 19:47:55 2019 +0530
+
+    Cheers!
```
Comment (nicks9188) [inline jhgkh]: Added anonymously. so will not be adding it.

File: newCommit.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+newcommit
```
Comment (nicks9188) [inline newCommit.txt:1]: To be added.

**Review by chachachaudharyy — CHANGES_REQUESTED** [review]
delete the comment file.

**Review by nicks9188 — COMMENTED** [review]
Done

**Review by chachachaudharyy — APPROVED** [review]

**Pull request merged**